spec/lib/govuk_component/inset_text_helpers_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe GovukComponent::InsetTextHelpers, type: :helper do
+  include RSpecHtmlMatchers
+
+  describe '#govuk_inset_text' do
+    subject(:markup) { helper.govuk_inset_text() { 'My content' } }
```

Comment:
in ruby the braces are optional and idiomatically not added unless args are need
```
 subject(:markup) { helper.govuk_inset_text { 'My content' } }
```

unless i am missing something?